service/src/test/java/com/epam/rft/atsy/service/passwordchange/validation/impl/PasswordUniqueRuleTest.java

```diff
@@ -0,0 +1,143 @@
+package com.epam.rft.atsy.service.passwordchange.validation.impl;
+
+import com.epam.rft.atsy.service.PasswordChangeService;
+import com.epam.rft.atsy.service.domain.PasswordChangeDTO;
+import com.epam.rft.atsy.service.security.UserDetailsAdapter;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Matchers;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.BDDMockito.given;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PasswordUniqueRuleTest {
+
+    @Mock
```

Comment:
Move this line above the @InjectMocks line. So they are together, and constants are defined first in the class.